src/tests/ftest/dfuse/container_attrs.py

```diff
@@ -0,0 +1,113 @@
+"""
+  (C) Copyright 2020-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from apricot import TestWithServers
+from dfuse_utils import get_dfuse, start_dfuse
+
+
+class DfuseContainerAttrs(TestWithServers):
+    """Check if the dfuse attributes of a container are properly managed.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # obtain separate logs
+        self.update_log_file_names()
+
+        # Start the servers and agents
+        super().setUp()
+
+    def check_attrs(self, dfuse, container_name):
```

Comment:
This is a helper function so should be "private" to the class.
```suggestion
    def _check_attrs(self, dfuse, container_name):
```

Reply:
> * [x]  Move check_attrs() to private.

Fixed with commit 865f2e549